src/app/pipes/record-status.pipe.ts

```diff
@@ -1,11 +1,25 @@
 import { Pipe, PipeTransform } from '@angular/core';
+import { RecordStatus } from 'src/app/enums/record-status.enum';
 
 @Pipe({ name: 'recordStatusDisplay' })
 export class RecordStatusDisplayPipe implements PipeTransform {
-    transform(value: number) {
-        if (value === 0) return '未提交';
-        if (value === 1) return '已提交';
-        if (value === 2) return '院系管理员已审核';
-        if (value === 3) return '学校管理员已审核';
+    transform(value: RecordStatus) {
```

Comment:
Hmm, it's better, but I think we can use a map to provide <status, human_readable_status> pairs, then we don't need the `switch` any more.